src/main/java/org/datadog/jmxfetch/reporter/StatsdReporter.java

```diff
@@ -3,11 +3,13 @@
 import com.timgroup.statsd.NonBlockingStatsDClientBuilder;
 import com.timgroup.statsd.ServiceCheck;
 import com.timgroup.statsd.StatsDClient;
-import com.timgroup.statsd.StatsDClientErrorHandler;
 import lombok.extern.slf4j.Slf4j;
 import org.datadog.jmxfetch.Instance;
 import org.datadog.jmxfetch.JmxAttribute;
 
+import java.util.concurrent.atomic.AtomicInteger;
```

Comment:
NIT: I don't this import is used in this context

Reply:
Thanks, bad cleanup from my end! :)